kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -135,7 +135,6 @@
       },
       footerLength() {
         return (
-          1 +
```

Comment:
Why was this update of the progress bar max width calculation needed to fix cards heights?

Reply:
From my understanding of what @rtibbles told me, there may have been a three-dots button for a menu in the footer that wasn't conditional, so the `1` was there to provide some space for that. However, that element was removed, but the `1` did not get removed.

Reply:
Ah okay, sounds like an additional cleanup not related to the height fix. Thank you.